tests/gate.test.ts

```
@@ -126,6 +128,50 @@ describe("gate", () => {
         expect(panes[0].t).not.toBeNull()
         expect(panes[0].d).not.toBeNull()
         expect(panes[0].d.resize).toHaveBeenCalledTimes(0)
+    })
+    it("stores a session hash on first empty webexec payload", async () => {
+        let g = t.addGate()
```

ESLint

[error] 'g' is never reassigned. Use 'const' instead.

(prefer-const)